mongo_connector/doc_managers/elastic_doc_manager.py

```diff
@@ -35,6 +35,8 @@
 from mongo_connector.doc_managers.doc_manager_base import DocManagerBase
 from mongo_connector.doc_managers.formatters import DefaultDocumentFormatter
 
+from requests_aws4auth import AWS4Auth
```

Comment:
Perhaps try importing this and catch ImportError. If `aws` is given in kwargs below and we failed to import requests_aws4auth, let's raise an Exception notifying the user to install `requests_aws4auth`.